helmfile/values/opensearch/dashboards.yaml.gotmpl

```diff
@@ -54,6 +54,9 @@ ingress:
     - host: {{ .Values.opensearch.dashboards.subdomain }}.{{ .Values.global.baseDomain }}
       paths:
         - path: /
+          backend:
+            serviceName: ""
+            servicePort: ""
```

Comment:
I still think that this is strange, shouldn't you need to set the correct service name and port here? Or are you sure it works without it?

Reply:
it seems to be working without it indeed. I had to double-check. Looks as if they made it work out of the default value rather than needing set ones: https://github.com/opensearch-project/helm-charts/blob/main/charts/opensearch-dashboards/values.yaml

Reply:
Make a note on why they are necessary to have specified.